rabbitmq/os_tweaks.sls

```diff
@@ -0,0 +1,11 @@
+{% from "rabbitmq/map.jinja" import rabbitmq with context %}
+
+increase_file_descriptor_limit:
+  cmd.run:
+    - name: sysctl -w fs.file-max={{ rabbitmq.fd_limit }}
+  file.append:
+    - name: /etc/sysctl.conf
+    - text: fs.file_max={{ rabbitmq.fd_limit }}
```

Comment:
Yes, this is because of the need to potentially have a large number of open network sockets and for when messages need to get paged to disk. http://www.rabbitmq.com/production-checklist.html